### PR TITLE
Add initial and final wait values

### PR DESCRIPTION
**Pull request by OdeclasV**
Also changed the success msg. For some reason, tests failed with the older message, even though it has not changed in the page (see below)
image

### CODE REVIEW

File: members/CTL000037.yaml [hunk 1]
```diff
@@ -36,8 +40,10 @@ contact_form:
...
         - value: Submit
           selector: form#webform-client-form-1797 div.form-actions input.webform-submit
+    - wait:
+        - value: 7
   success:
     headers:
       status: 200
     body:
-      contains: Thank you for your message.
+      contains: Go back to the form
```
Comment (annanhan) [inline members/CTL000037.yaml]: does this message ONLY appear if the form submits successfully?

Comment (OdeclasV) [reply members/CTL000037.yaml]: I believe so. It's in the thank you page supporters get once the msg is submitted (see image below). I don't see the 'Go back to form' message anywhere in the page. 

image

Comment (annanhan) [reply members/CTL000037.yaml]: I see "Thank your for your message." there. Why did you change it?

**Comment by OdeclasV** [comment]
@annanhan success msg has been changed

**Pull request merged**